threat_intel/__init__.py

```diff
@@ -253,9 +230,9 @@
 }
 
 
-URL reports
------------
-Calls url/report VirusTotal API endpoint.
+#### URL report Endpoint
```

Comment:
Capitalization should be more consistent throughout the README. Either all words start from the big cap, like in titles, or all words should be starting from the small cap (unless they are some specific terms, like "URL").